external/concealer/src/autoware_universe.cpp

```diff
@@ -109,7 +109,7 @@ auto AutowareUniverse::engage() -> void
     waitForAutowareStateToBeDriving([this]() {
       auto request = std::make_shared<Engage::Request>();
       request->engage = true;
-      requestEngage(request);
+      requestEngage(request, 10);
```

Comment:
Please add a comment as to why the number "10" is given.
In other words, please add the fact that "We attempt to resend the service up to 10 times, but this number of times was determined by heuristics, not for any technical reason.